config/sections/mixins/layout.php

```diff
@@ -133,8 +141,10 @@
 				$item['info'] = $model->toString($this->info);
 			}
 
-			// Use form to get the proper values for the columns
-			$form = Form::for($model)->values();
+			if ($this->rawvalues !== true) {
```

Comment:
I was wondering if we could turn this into two separate code blocks for the two variants for better clarity. The combination of the conditional $form variable with the match clause in the foreach loop is a bit hard to parse.